README-en.md

```diff
@@ -0,0 +1,172 @@
+
+When asking and answering questions, avoid any imperative or impatient language, and be cordial, after all we are all here to help one another!
+
+After posting your question, be sure to look for alternatives to solve your problem. Send new messages with updates and alternatives you've tried. If you can solve your problem on your own, spread the knowledge you have gained by describing the solution.
+
+In no situation repeat the same message several times. In addition to generating spam, the community will show much more interest in helping those who are trying to advance on their own.
+
+Avoid messages without content, merely to agree to previous messages, loose words or excessively long messages without need. Use the options that Slack offers, such as the Reactions option. Also considered messages without content questions such as: Has anyone used the XYZ tool? or someone can help me ?. In case of doubts, problems or other situations, directly send your question or situation in the correct channel, avoiding to pollute the channels with messages without content. In addition to these cases, complaints without constructive purpose and for mere outburst are also messages without content. Remember that multiple channels have thousands of participating members, so value the channels and the space to exchange information avoiding messages without purpose.
+
+### Be kind
+
+When responding, asking or commenting, avoid aggressive or disproportionate tone messages. Remember that our group is open and highly diversified. We have members with extensive experience in application development as well as beginners who have just met the platform. We also have different opinions and visions on almost every subject. And that's what makes Android Dev BR a great place to learn and develop.
+
+Pay attention to your posts to keep communication enjoyable and interesting for everyone.
+
+### Individual responsibility
+
+All messages sent in Slack, through public or private means, are the sole responsibility of the author. Android Dev BR enhances the communication between developers from Brazil and the World, but is not responsible for published information or opinions. All members are free to submit information and opinions that they deem most appropriate, desde que não atentem contra a legislação brasileira ou contra este Código de Conduta. Any measures around messages should be applied to the author of the message. AndroidDevBR is not responsible for sensitive information shared on channels or private messages. When sending snippets of code, be careful not to send credentials (endpoints, API keys, and the like). If any of this information is shared, immediately notify any administrator to have the message deleted.
```

Comment:
Sobrou um trecho em português: `desde que não atentem contra a legislação brasileira ou contra este Código de Conduta`

Reply:
_given no Brazilian Law or Code of Conduct rule is violated_